risc0/build/src/lib.rs

```diff
@@ -159,16 +159,17 @@ pub fn embed_methods() {
                     eprintln!("Try: `cargo run --bin risc0-build-methods`");
                     std::process::exit(-1);
                 }
-                let method_id = MethodID::new(&elf_path.to_str().unwrap()).unwrap();
-                method_id.write(&id_path.to_str().unwrap()).unwrap();
+                let method_id: &[u8];
+                unsafe {
```

Comment:
Why is this unsafe?

Reply:
Good call :)

Reply:
I think it's just leftover from whatever was there before. As a rust noob Im doing a lot of experimentation...This stuff will be cleaned up before I remove the draft status.